keps/sig-apps/592-ttl-after-finish/kep.yaml

```diff
@@ -5,18 +5,47 @@ authors:
 owning-sig: sig-apps
 participating-sigs:
   - sig-api-machinery
+status: implemented
```

Comment:
implementable

[status should be changed to implemented after GA graduation]

Reply:
done